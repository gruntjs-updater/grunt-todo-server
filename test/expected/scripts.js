var TODO_DATA = {"test/fixtures/scripts.js":[{"raw":"TODO: Test","prefix":"todo","comment":" Test"},{"raw":"TODO: Test 2","prefix":"todo","comment":" Test 2"}]};