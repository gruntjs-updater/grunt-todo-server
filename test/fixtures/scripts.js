
var a = 1;
var b = 2;
var c = 3;

/* TODO: test */

function foo() {
  console.log(a);
}

/* TODO: 2nd */
function bar() { }