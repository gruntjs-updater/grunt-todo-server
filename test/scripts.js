'use strict';

var grunt = require('grunt');

exports.todo_server = {
  setUp: function(done) {
    done();
  },

  scripts: function(test) {
    test.expect(1);

    var actual = grunt.file.read('server/index.html');
    var expected = grunt.file.read('test/expected/scripts.html');

    test.equal(actual, expected, 'the extracted todos should match');
    test.done();
  }
};
