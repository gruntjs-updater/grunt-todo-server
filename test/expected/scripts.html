{"test/fixtures/scripts.js":[{"raw":"TODO: test */","prefix":"TODO","comment":" test */"},{"raw":"TODO: 2nd */","prefix":"TODO","comment":" 2nd */"}]}